conf/pipeline/sarek/uppmax.config

```diff
@@ -3,10 +3,12 @@
 params {
   config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
   config_profile_description = 'nf-core/sarek uppmax profile provided by nf-core/configs'
-}
 
-params {
-  // Just useful until iGenomes is updated on UPPMAX
+// Just useful until iGenomes is updated on UPPMAX
   igenomeIgnore = true
   genomes_base = params.genome == 'GRCh37' ? '/sw/data/uppnex/ToolBox/ReferenceAssemblies/hg38make/bundle/2.8/b37' : '/sw/data/uppnex/ToolBox/hg38bundle'
 }
+
+if (hostname ==~ "i.*") {
```

Comment:
I think you should add this for hostname r.* and b.* as well?

Reply:
If I remember well `r.*` and `b.*` have a similar settings, I'll check what we used to give with the old scilifelab/sarek repo

Reply:
we had that:
https://github.com/SciLifeLab/Sarek/blob/master/conf/uppmax-slurm.config#L13

Reply:
Might want to have that in Megabytes to be sure that you avoid problems with the GB to MB conversion.

Reply:
I think that's why we have 7 and not 8 GB